app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
+  propTypes: {
+    comment: React.PropTypes.object,
+    level: React.PropTypes.number,
+  },
+
+  getDefaultProps: function() {
+    return {
+      level: 0,
+    };
+  },
+
+  render() {
+    let comment = this.props.comment;
+    let replies = comment.replies;
+    let repliesLength = replies ? replies.length : 0;
+    let commentBoxClasses = 'comment__box ' + (this.props.level % 2 === 0 ? 'comment__box-even' : 'comment__box-odd');
+
+    return (
+      <div className={commentBoxClasses}>
+
+        <div className="comment__arrows">
+          <div className="comment__vote-arrow"><FontAwesome name="arrow-up" /></div>
+          <div className="comment__upvote-points">{comment.upvotes}</div>
+        </div>
+
+        <div className="comment__contents">
+          <div>
+            <span className="comment__name">{comment.name}</span> -
+            <span className="comment__delta">{comment.delta}</span>
+          </div>
+          <div className="comment__message">{comment.message}</div>
+
+          <If condition={repliesLength}>
+            {replies.slice(0, repliesLength).map((reply) => (
```

Comment:
Also, I recommend putting this a `renderReplies_()` method for separation of concerns.